flake.nix

```diff
@@ -18,6 +18,9 @@
     in
     {
       lib = lib.extend (final: prev: {
+
+        nixos = import ./nixos/lib { inherit final; };
```

Comment:
should this be?
```suggestion
        nixos = import ./nixos/lib { lib = final; };
```

Reply:
Thanks so much! I missed this in my manual testing. I used `...` for forward compatibility, but this is a price to pay. Hmm...